lib/infraestructure/model/task.dart

```diff
@@ -1,17 +1,30 @@
 import 'package:healthy_routine_mobile/healthy_routine.dart';
 
 class Task {
+  final int id;
   final String name;
   final TaskStatus status;
   final List<Weekday> recurrence;
   final DateTime startDate;
   final DateTime endDate;
 
   Task({
+    this.id,
     this.name,
     this.status,
     this.recurrence,
     this.startDate,
     this.endDate,
   });
+
+  // Convert a Note object into a Map object
+  Map<String, dynamic> taskMap() {
```

Comment:
I think `asMap()` would be a better name, to remind a cast to map

Reply:
Done!